configuration/internal.go

```diff
@@ -234,3 +234,9 @@ type updateTagRequest struct {
 	Name     string `json:"name"`
 	GroupIDs []int  `json:"group_ids"`
 }
+
+type listGroupsPropertiesRequest struct {
+	GroupIDs   []int  `json:"group_ids,omitempty"`
+	Namespace  string `json:"namespace,omitempty"`
+	NamePrefix string `json:"namespace_prefix,omitempty"`
```

Comment:
Copied from internal codebase:

```go
type ListGroupsPropertiesRequest struct {
	NamePrefix      string  `json:"name_prefix"`
}
```

you need to be consistent with fields naming.

Reply:
Fixed. Also double-checked for other instances of namespace_prefix introduced by me, but none was found.